Apps/Common/scripts/iotest.sh.in

```diff
@@ -16,16 +16,21 @@ mysim=$1
 
 cd `dirname $2`
 MAPFILE=`head -n1 $2`
+tmplog=`mktemp -t ifemlogXXXXXX`
 tmpnew=`mktemp -t ifemlogXXXXXX`
 tmpold=`mktemp -t ifemlogXXXXXX`
 output=`mktemp -t ifemXXXXXX.$3`
 if [ "$3" == "hdf5" ]
 then
-  $mysim $MAPFILE -hdf5 $output 2>&1 > /dev/null
+  $mysim $MAPFILE -hdf5 $output 2>&1 > $tmplog
```

Comment:
Can we also use `| tee $tmplog` instead? (same also in line 27).

Then a `ctest --verbose` will produce all console output from the IFEM app as well, which I find useful to track what's going on when the regression test passes, but not when I try to run it manually outside ctest.
Or will this slow down or disturb the jenkins process?

Similar request also for `regtest.sh`